inference-engine/tests/functional/plugin/shared/include/execution_graph_tests/dump_constant_layers.hpp

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <pugixml.hpp>
+#include "gtest/gtest.h"
+
+namespace ExecutionGraphTests {
+
+class ExecGraphDumpConstantLayers : public testing::TestWithParam<std::string> {
```

Comment:
I think you have already discussed it but just wondering if we really need tests for this debug feature?

Reply:
I think we need tests for this debugging feature.
@dmitry-gorokhov what do you think?

Reply:
This test is intended to check that exec_graph doesn't contain const nodes by default. Executable graph - public funtionality which we need to validate.
Tests for debug capabilities - separate topic. Lets discuss not in bounds of this PR.

Reply:
Basically the question about the test for the PR is not applicable anymore, since now it is default behavior (to not include constant nodes) and I agree that this better to be tested.